src/python/pants/engine/subsystem/native_engine_version

```diff
@@ -1 +1 @@
-e5b11346a458240934cc6136e40faa62b75f79ff
+da39a3ee5e6b4b0d3255bfef95601890afd80709
```

Comment:
Yeah, I saw a lot of errors in CI. I believe they are due to this. I don't know why this got changed, but I think I should change it back, right?